Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: catalogs/databricks/README.md

  - name: databricks:<CATALOG_NAME>
    from: db_uc
    params:
      mode: spark_connect # or delta_lake
      databricks_token: ${env:DATABRICKS_TOKEN}
      databricks_endpoint: <instance-id>.cloud.databricks.com
      databricks_cluster_id: <cluster-id>
```

For `mode` you can choose between `spark_connect` or `delta_lake`. `spark_connect` is the default mode and requires an [All-Purpose Compute Cluster](https://docs.databricks.com/en/compute/index.html) to be available. `delta_lake` mode queries directly against Delta Lake tables in object storage, and requires Spice to have the necessary permissions to access the object storage directly.

> The `DATABRICKS_TOKEN` environment variable should be set to the Databricks personal access token created in Step 1. You can create a `.env` file in the same directory as `spicepod.yaml` and set the environment variable there, i.e.:

Reviewer: ```suggestion
Set the `DATABRICKS_TOKEN` environment variable to the Databricks personal access token created in Step 1. A `.env` file created in the same directory as `spicepod.yaml` can be used to set the variable, i.e.:
```

  
```bash
echo "DATABRICKS_TOKEN=<token>" > .env
```

Visit the documentation for more information configuring the [Databricks Unity Catalog Connector](https://docs.spiceai.org/components/catalogs/databricks).

## Step 4. Set the object storage credentials for `delta_lake` mode

When using the `delta_lake` mode, the object storage credentials must be set for Spice to access the data.

### Using Delta Lake directly against AWS S3

```yaml
params:
  mode: delta_lake
  databricks_token: ${env:DATABRICKS_TOKEN}
  databricks_aws_access_key_id: ${env:AWS_ACCESS_KEY_ID}
  databricks_aws_secret_access_key: ${env:AWS_SECRET_ACCESS_KEY}

>   databricks_aws_region: <region>

Reviewer: Can we provide examples in comments?

>   databricks_aws_region: <region> # i.e. us-east-1, us-west-2

Reviewer: ```suggestion
  databricks_aws_region: <region> # E.g. us-east-1, us-west-2
```
